edsby.py

```diff
@@ -190,7 +190,13 @@ def getauthData(self, loginData):
         student metadata returned by Edsby
     """
     def sendAuthenticationData(self):
-        studentData = requests.post('https://'+self.edsbyHost+'/core/login/'+str(self.instanceMeta['nid'])+'?xds=loginform&editable=true',data=self.authData,cookies=self.getCookies(),headers=self.getHeaders()).json()
+        studentData = requests.post('https://'+self.edsbyHost+'/core/login/'+str(self.instanceMeta['nid'])+'?xds=loginform&editable=true',data=self.authData,cookies=self.getCookies(),headers=self.getHeaders())
+        nookies = {
+            'session_id_edsby': dict(studentData.cookies)['session_id_edsby'],
+            '__cfduid': dict(self.getCookies())['__cfduid']
```

Comment:
Side note- this can be removed in future revisions, as the __cfduid cookie is [being deprecated](https://blog.cloudflare.com/deprecating-cfduid-cookie/).